package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
-    "start": "webpack-dev-server --open"
+    "start": "webpack-dev-server -d --open"
```

Comment:
Does this cause the dev server to be detached? This is a personal preference option, so no need to push it onto everybody else (me)

Reply:
My assumption was incorrect.